src/web/server/locales/cy/common.json

```diff
@@ -99,11 +99,10 @@
     "weJustSentASixDigitCodeTo": "Lorem ipsum 6 dolor sit amet",
     "itMayTakeAFewMinutesToArrive": "Do eiusmod tempor incididunt.",
     "codeLabel": "Oced",
-    "requestANewCode": "Lorem ipsum dolor",
-    "ifItDoesNotArrive": "sunt in culpa qui officia."
+    "requestNewCodeLink": "<a href=\"/send-code\" id=\"{{requestNewCodeLinkId}}\">Consectetur adipiscing elit</a> consectetur adipiscing elit"
```

Comment:
Why does the ID need to be a variable?

Reply:
So it's not repeated across the translation files.